CONTRIBUTING.md

```diff
@@ -621,28 +649,28 @@ The symbolserver over at https://download.arangodb.com/symsrv_arangodbXX/ is
 browseable; thus you can easily download the files you need by hand. It
 contains of a list of directories corresponding to the components of ArangoDB, e.g.:
 
-  - arango - the basic arangodb library needed by all components
-  - arango_v8 - the basic V8 wrappers needed by all components
-  - arangod - the server process
-  - the client utilities:
-    - arangobackup
-    - arangobench
-    - arangodump
-    - arangoexport
-    - arangoimport
-    - arangorestore
-    - arangosh
-    - arangovpack
+- arango - the basic arangodb library needed by all components
```

Comment:
make it lib/ pls.

Reply:
Hello @dothebart, I just pushed an update to fix this.